.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
```

Comment:
Should there be a pull request limit? This could keep it from suggesting new dependencies, even security-related ones.

Reply:
No need for the bot to put in the work if the PRs aren't being merged. New PRs will open as needed once the first 10 are merged.